corehq/apps/reports/const.py

```diff
@@ -2,6 +2,7 @@
 
 USER_QUERY_LIMIT = 5000
 DEFAULT_PAGE_LIMIT = 10
+EXPORT_PAGE_LIMIT = 5000
```

Comment:
My instinct would be to set this to around 1000 without more context, FWIW. Already 100x the current amount, so if that doesn't help, it's unlikely 5000 will. I can't say I have a strong sense of where the right balance, so happy to defer to you if you're confident from past trials that 5000 tends to be a better balance

Reply:
When we were testing today, 10 and 1,000 always timed out (number of attempts for 1,000 was low) while 10,000 worked every time (also low number of attempts) and 5,000 worked intermittently (tried several times). All those attempts were done on a particular problematic report with >20K rows. Another report with a similar number of rows was working just fine with a page size of 10. Those are very small sample sizes, obviously, so probably worth doing more investigation.

As a point of reference, the default scroll page size is 1,000 when no page size is provided. Not sure when that is used, but it's definitely not used for the type of report we were testing.

https://github.com/dimagi/commcare-hq/blob/1346a3d261e52956140326f6dbede889887689a8/corehq/apps/es/const.py#L12

Reply:
Okay, seems like you have reason to believe 5,000 may be better than 1,000, and doesn't seem worse, so that's good enough for me!